src/crimson/os/seastore/transaction.h

```diff
@@ -488,13 +503,16 @@ class Transaction {
    */
   io_stat_t fresh_block_stats;
   uint64_t num_delayed_invalid_extents = 0;
+  uint64_t num_allocated_invalid_extents = 0;
   /// blocks that will be committed with journal record inline
   std::list<CachedExtentRef> inline_block_list;
   /// blocks that will be committed with out-of-line record
-  std::list<CachedExtentRef> ool_block_list;
+  std::list<CachedExtentRef> written_ool_block_list;
   /// blocks with delayed allocation, may become inline or ool above
   std::list<LogicalCachedExtentRef> delayed_alloc_list;
 
+  std::list<LogicalCachedExtentRef> allocated_ool_block_list;
```

Comment:
Add comment:
/// blocks that are allocated but not written, will become ool above